src/BenchmarksApps/TodosApi/appsettings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "Authentication": {
+    "Schemes": {
+      "Bearer": {
+        "ValidAudiences": [
+          "http://localhost:5054"
```

Comment:
Is there a way that `5054` is always used? I thought that was in the launchsettings.json or something?

Reply:
Right, I copied it from launchsettings.json for this app, but I should likely add http://localhost:5000 to cover the published case too.